SparkPost/2web2ui
# FE-135: Reinstate gtm.js tag manager event

This is a minimal patch to our Google Tag Manager integration to emit a `gtm.js` event. This event produces the so-called "Page View" event in Tag Manager and our marketing folks have various tracking actions triggered by that it.

[Some background material](https://www.simoahava.com/analytics/trigger-guide-google-tag-manager/)

### Testing

All GTM events are pushed onto a global named `dataLayer` which you can review in the console.
1. Log in to the app
1. Verify that the `gtm.js` event is listed on the `dataLayer` variable.

You can also do this in production to verify that the event is missing.
